### ISSUE TITLE
Tutorial to use with nuxt.js?

### ISSUE THREAD

**Issue by rlam3** [alfonsobries/vue-tailwind#10]
Tutorial to use with nuxt.js?

**Comment by alfonsobries** [comment]
I will add that to the docs but in the meantime you just need to create a file like

`plugins/vue-tailwind.js`

with the following content

```
import Vue from 'vue';
import VueTailwind from 'vue-tailwind';

Vue.use(VueTailwind, {
       // optional theme (according to the docs)
	theme: {
		TCard:{
		  baseClass: 'bg-white rounded-lg shadow',
		  headerClass: 'px-12 pt-8 pb-0 text-xl',
		  bodyClass: 'p-12',
		},
      }
});
```

Then in the `nuxt.config.js` add the plugin in the plugins array

```
module.exports = {
  //...
  plugins: [
    // ... 
    '~plugins/vue-tailwind',
  ],
  //...
}
```

That should be enough to register the components

**Comment by alfonsobries** [comment]
In this project I use nuxt with this package, maybe its useful for you https://github.com/alfonsobries/laravel-nuxt-client

**Comment by alfonsobries** [comment]
I will close this due inactivity

**Comment by sergiooak** [comment]
Sorry to reopen it, but I'm facing a wierd problem...

New components works just fine with this tutorial (Datepicker, Toggle, etc) but older ones works only partial, all html and JS works, but it not any classes.

1 - Plugin and Import
![image](https://user-images.githubusercontent.com/46563926/96919302-f80efa80-1481-11eb-9254-e81f40b0363e.png)
![image](https://user-images.githubusercontent.com/46563926/96919389-12e16f00-1482-11eb-837b-d4dfd6e99c21.png)

2 - Using new components
![image](https://user-images.githubusercontent.com/46563926/96919617-65bb2680-1482-11eb-89ff-0026ff67b812.png)
![image](https://user-images.githubusercontent.com/46563926/96919610-618f0900-1482-11eb-8533-686281265043.png)
![image](https://user-images.githubusercontent.com/46563926/96919668-7bc8e700-1482-11eb-8081-5f2db21dcdb0.png)

3 - using "older" components
![image](https://user-images.githubusercontent.com/46563926/96919727-9307d480-1482-11eb-8c45-7a83460706d1.png)
![image](https://user-images.githubusercontent.com/46563926/96919756-9c913c80-1482-11eb-97d5-7661deb1779e.png)
![image](https://user-images.githubusercontent.com/46563926/96919827-b6cb1a80-1482-11eb-9faf-b0712a138569.png)

---

So basically vanilla components are not rendering any class
